src/lsqecc/pauli_rotations/rotation.py

```diff
@@ -192,6 +229,15 @@ def from_list(pauli_ops: List[PauliOperator], rotation: Fraction) -> "PauliRotat
             r.change_single_op(i, op)
         return r
 
+    def get_y_free_equivalent(self) -> List["PauliRotation"]:
+        """Return the equivalent of current Pauli Rotation but without Y operator.
+        Only supports pi/8 rotations at the moment.
+        """
+        if self.rotation_amount not in (Fraction(1, 8), Fraction(-1, 8)):
```

Comment:
This check is irrelevant because the remove operation stands true for all angles. and any Y operator can be removed